scripts/deploy_protocol.py

```diff
@@ -0,0 +1,106 @@
+import os
+import logging
+import json
+from asyncio import run
+from starknet_py.net.account.account_client import (AccountClient)
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+from starknet_py.contract import Contract
+from starknet_py.net.models import StarknetChainId
+from starkware.crypto.signature.signature import private_to_stark_key
+from starknet_py.net.gateway_client import GatewayClient
+from utils import declare_and_deploy_contract, declare_contract
+from pathlib import Path
+from dotenv import load_dotenv
+
+#Logging conf, just want to print results to console
+logging.basicConfig(level = logging.INFO)
+
+# Address of the ETH token on mainnet,testnet1 and testnet2
+ETH_ADDRESS = 2087021424722619777119509474943472645767659996348769578120564519014510906823
+# Set a high max fee for the deployments
+MAX_FEE = int(1e16)
+
+# Loading .env file
+dotenv_path = os.path.join('./', '.env')
+load_dotenv(dotenv_path)
+
+# Get env variables
+private_key = int(os.environ.get("PRIVATE_KEY"))
+account_address = int(os.environ.get("ACCOUNT_ADDRESS"),16)
+network = os.getenv('NETWORK')
+
+#Current starknet.py version does not support testnet2 as a default network
+if network == "testnet2":
+    network = "https://alpha4-2.starknet.io"
+
+#Configure Admin AccountClient
+public_key = private_to_stark_key(private_key)
+signer_key_pair = KeyPair(private_key,public_key)
+client = AccountClient(address=account_address, client=GatewayClient(net=network), key_pair=signer_key_pair, chain=StarknetChainId.TESTNET, supported_tx_version=1)
+#Get Kakarot ABI
+file = open("build/kakarot_abi.json")
+kakarot_abi = json.load(file)
+file.close()
+
+async def main():
+
+    logging.info("----------------------------------")
+    logging.info("--- Deploying Kakarot Protocol ---")
+    logging.info("----------------------------------")
+    logging.info(".")
+    logging.info(".")
+    logging.info(".")
+
+    #################################
+    #                               #
+    #   DECLARE & DEPLOY CONTRACTS  #
+    #                               #
+    #################################
+    
+
+    # Declare EVM Contract
+    logging.info("⏳ Declaring EVM Contract Account... ")
+    evm_account_class_hash = await declare_contract(client,Path("./build/", "contract_account.json").read_text("utf-8"))
```

Comment:
you could create a `BUILD_DIR = Path("build")` outside of the `main` fun. Also, the advantage of pathlib is to write `Path("build") / "contract_account.json"` to let them put the `/` according to the OS.

Reply:
Implemented, thanx for explaining it :)
I tend to forget others work on different machines.